cpp/src/spatial/hausdorff.cu

```diff
@@ -80,18 +85,23 @@ struct hausdorff_functor {
                                            result_view.begin<T>(),
                                            stream);
 
-    return result;
+    thrust::host_vector<cudf::size_type> splits(num_spaces - 1);
+    thrust::sequence(thrust::host, splits.begin(), splits.end(), num_spaces, num_spaces);
```

Comment:
Could this be on the device?

Reply:
No, `cudf.split` takes `host_span` as its argument.

Reply:
But do we really need to use cudf::split to do this operation? It's all just views, right?

Reply:
That's true, we could hand-roll our split operation here. cuspatial's column doesn't contain any null, so we don't need to do null counting on device.

Reply:
OK. Just trying to avoid calling cuDF detail API...